Packs/TheHiveProject/Integrations/TheHiveProject/TheHiveProject.py

```diff
@@ -488,7 +486,7 @@ def update_case_command(client: Client, args: dict):
     # Get the case first
     original_case = client.get_case(case_id)
     if not original_case:
-        return_error(f'Could not find case ID {case_id}')
+        raise Exception(f'Could not find case ID {case_id}.')
```

Comment:
do we have a reason for the failure? if so, it should be added to the logs.
```suggestion
        raise DemistoException(f'Could not find case ID {case_id}.')
```